pkg/webui/console/views/device-add/messages.js

```diff
@@ -16,7 +16,7 @@ import { defineMessages } from 'react-intl'
 
 const messages = defineMessages({
   // Shared messages.
-  repositoryTabTitle: 'From The Things Device Repository',
+  repositoryTabTitle: 'From The LoRaWAN Device Repository',
```

Comment:
@laurensslats decided that it should be called `The Things Device Repository`, so maybe the title in the DR should be changed rater? cc @johanstokking

Reply:
We need a decision here @johanstokking

Reply:
Why was this marked as resolved, @bafonins?